src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/messaging/PrisonerReceivedDueToAdmissionEventTest.kt

```diff
@@ -0,0 +1,233 @@
+    assertThat(inductionScheduleEvent.personReference.identifiers[0].value).isEqualTo(prisonNumber)
+    assertThat(inductionScheduleEvent.detailUrl).isEqualTo("http://localhost:8080/inductions/$prisonNumber/induction-schedule")
+  }
+
+  @Test
+  fun `should update the Review Schedule and send outbound message given 'prisoner received' (admission) event for prisoner that already has an Induction and initial Review Schedule`() {
+    // Given
+    // an induction schedule, induction and action plan are created. This will have created the initial Review Schedule
+    val prisonNumber = randomValidPrisonNumber()
+    createInductionSchedule(
+      prisonNumber = prisonNumber,
+      status = InductionScheduleStatus.COMPLETED,
+    )
+    createInduction(prisonNumber, aValidCreateInductionRequestForPrisonerNotLookingToWork())
+    createActionPlan(prisonNumber)
+
+    // The above calls set the data up but they will also generate events so clear these out before starting the test.
+    // Before clearing the queues though we need to wait until the first "plp.review-schedule.updated" event on the REVIEW queue is received.
+    await untilCallTo {
+      reviewScheduleEventQueue.countAllMessagesOnQueue()
+    } matches { it != null && it > 0 }
+    clearQueues()
+
+    val expectedPrisoner = aValidPrisoner(prisonerNumber = prisonNumber)
+    createPrisonerAPIStub(prisonNumber, expectedPrisoner)
+
+    val earliestDateTime = OffsetDateTime.now()
+
+    val sqsMessage = aValidHmppsDomainEventsSqsMessage(
+      prisonNumber = prisonNumber,
+      eventType = PRISONER_RECEIVED_INTO_PRISON,
+      additionalInformation = aValidPrisonerReceivedAdditionalInformation(
+        prisonNumber = prisonNumber,
+        reason = ADMISSION,
+      ),
+    )
+
+    // When
+    sendDomainEvent(sqsMessage)
+
+    // Then
+    // wait until the queue is drained / message is processed
+    await untilCallTo {
+      domainEventQueueClient.countMessagesOnQueue(domainEventQueue.queueUrl).get()
+    } matches { it == 0 }
+
+    val actionPlanReviews = getActionPlanReviews(prisonNumber)
+    assertThat(actionPlanReviews)
+      .latestReviewSchedule {
+        it.hasStatus(ReviewScheduleStatus.SCHEDULED)
+          .hasCalculationRule(ReviewScheduleCalculationRule.PRISONER_READMISSION)
+          .wasCreatedAtOrAfter(earliestDateTime)
+          .wasUpdatedAtOrAfter(earliestDateTime)
+      }
+
+    // test that outbound event is also created:
+    val reviewScheduleEvent = inductionScheduleEventQueue.receiveEvent(QueueType.REVIEW)
+    assertThat(reviewScheduleEvent.personReference.identifiers[0].value).isEqualTo(prisonNumber)
+    assertThat(reviewScheduleEvent.detailUrl).isEqualTo("http://localhost:8080/reviews/$prisonNumber/review-schedule")
+  }
+
+  @Test
+  fun `should re-schedule the Induction Schedule and send outbound message given 'prisoner received' (admission) event for prisoner that already has an incomplete Induction Schedule`() {
```

Comment:
This test method (and the next one) are the 2 new tests I've added to support the changes in this PR